modules/@apostrophecms/page-type/index.js

```diff
@@ -154,23 +154,6 @@ module.exports = {
           }
         }
       },
-      afterPublish: {
```

Comment:
Is this related to the in-context discard draft change? If not, it's a confusing inclusion.

Reply:
I believe I needed this fix to get the other to work.

This code was removed because moving a page now automatically reflects in the other mode, so this "replay it on publish" code is redundant, and also harmful.